Source/Tests/TrackableEntities.EF.5.Tests/FamilyModels/Parent.cs

```diff
@@ -13,6 +13,9 @@ public Parent(string name)
 
         [Key]
         public string Name { get; set; }
+
+        public string Nickname { get; set; }
```

Comment:
Here's a git hint: If you notice you forgot to add something to the previous commit.  You can make the change and simply [amend the last commit](https://help.github.com/articles/changing-a-commit-message/).

Don't be afraid to do a `git push --force` of your branch.  Because it's a PR, it won't affect anything else.